homeassistant/components/netatmo/__init__.py

```diff
@@ -144,6 +149,11 @@
 
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
     """Unload a config entry."""
+    if CONF_WEBHOOK_ID in entry.data:
+        await hass.async_add_executor_job(
+            hass.data[DOMAIN][entry.entry_id][AUTH].dropwebhook()
```

Comment:
Leave it up to the executor to call the function.

```suggestion
            hass.data[DOMAIN][entry.entry_id][AUTH].dropwebhook
```